### PR TITLE
Add photo to the readme

### LINKED ISSUE #1: README needs some flare
Maybe a photo of :tokyo_tower:?

@dice @lizclink Do you know of any good photos we can use?

### PR DESCRIPTION
**Pull request by johndbritton**
I really like this photo and think we should put it front and center, what do you think @lizclink?

Fixes #1 

:camera:

### CODE REVIEW

**Comment by lizclink** [comment]
had an inspiration @johndbritton!
tokyo-tower

File: README.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Introduction to Git & GitHub
 
+![dice and defunkt](https://cloud.githubusercontent.com/assets/2566204/8033835/d4ea459e-0e1c-11e5-8d1e-1214043e824f.JPG)
```
Comment (johndbritton) [inline README.md]: This needs a better caption!

Comment (lizclink) [inline README.md]: how about:
> the famous Tokyo skyline

**Comment by sawaka** [comment]
@johndbritton  how about this? 11391782_854009304634157_4619431382253666794_n

**Pull request merged**